articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -0,0 +1,178 @@
+
+> [!div renderon="docs"]
+> ## Register your application and download your quickstart app
+> You have two options to start your quickstart application:
+> * [Express] [Option 1: Register and then download your quickstart application](#option-1-register-your-application-and-download-your-quickstart-app)
+> * [Manual] [Option 2: Register your application and configure the sample](#option-2-register-and-manually-configure-your-application-and-code-sample)
+>
+> ### Option 1: Register your application and download your quickstart app
+>
+> 1. Go to the [Azure portal - Application Registration [Prod]](https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/JavascriptSpaQuickstartPage/sourceType/docs).
+> 1. Enter a name for your application and click **Register**.
+> 1. Follow the instructions to download and automatically configure your new application for you in one click.
+> [!div renderon="docs"]
+> ### Option 2: Register and manually configure your application and code sample
+> #### Step 1: Register your application
+> 
+> 1. To register an application, go to the [Azure portal - Application Registration [Prod]](https://aka.ms/registeredappsprod) and select **New registration**.
+> 1. Enter a name for your application, add `http<span/>://localhost:30662/` in **Reply URL**, and click **Register**.
+> 1. Select **Authentication** menu, set **ID tokens** under **Implicit Grant**, and then select **Save**.
+
+> [!div class="sxs-lookup" renderon="portal"]
+> ### Step 1: Configure your application in Azure portal
+> For the code sample for this quickstart to work, you need to add a reply URL as `http<span/>://localhost:30662/` and enable **Implicit Grant**.
+> > [!div renderon="portal" id="makechanges" class="nextstepaction"]
+> > [Make these changes for me]()
+>
+> > [!div id="appconfigured" class="hidden"]
+> > ![Already configured](media/active-directory-windesktop/checkmark.png) Your application is configured with these attributes
+
+> [!NOTE]
+> If you use Visual Studio, the Redirect URL will be set to to *http:<span/>//localhost:30662/* as it is configured in the code sample's project. If you use Python or any other web server, set the redirect URL to `http://<span/>localhost:30662/`, run the following command line:
+> ```bash
+> python -m http.server 30662
+> ```
+
+### Step 2: Download your web server or project
+
+- [Download the Visual Studio project](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/VisualStudio.zip)
+- [Download the core project files - for a local web server, such as Python](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/core.zip)
+
+### Step 3: Configure your JavaScript SPA
+
+> [!div renderon="docs"]
+> Edit `msalconfig.js` and replace `Enter_the_Application_Id_here` with the Application ID your application:
```

Comment:
@andretms Related to Allen's comment about msalconfig above. I think msalconfig is acceptable file name since MSAL is used in this sample. However, I agree that having some context on MSAL before this step will help. So I suggest in the intro lines for this quickstart we mention that the MSAL.js library is used in the sample.